src/components/PagesComponent/FrequentlyAskedQuestionsHealthCare/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
```

Comment:
no need to add faq in separate components. you can directly add in faqs section in page